client/src/app.js

```diff
@@ -334,6 +338,11 @@ export default function main({ DOM, HTTP, route, storage, scanner: scan$, search
                               ? { category: 'asset-txs',  method: 'GET', path: `/asset/${d.asset_id}/txs/chain/${last(d.last_txids)}` }
                               : { category: 'asset-txs',  method: 'GET', path: `/asset/${d.asset_id}/txs` }])
 
+    // Fetch Asset Icons                             
+    , !process.env.ASSET_MAP_URL ? O.empty() : O.of(
+      { category: 'asset-icons',  method: 'GET', path: `https://assets.blockstream.info/icons.json` }) 
```

Comment:
I'm not sure if loading all the icons all the time is the right way to go at this. The green apps do this to enhance privacy, so the server can't learn which assets the user is interested in. But in a block explorer all privacy bets are off anyway, so we could just make them available as proper images that could be embedded via a URL, say `https://assets.blockstream.info/icons/<assetid>.png`.

Loading images individually would help reduce bandwidth. The `icons.json` file currently weights 1MB and is expected to continue growing. And it would also avoid base64 encoding, which requires 1.37x more bytes.

There was some discussion at https://github.com/Blockstream/asset_registry/issues/26 about revamping the icons system, making them available as standalone files could be done as part of that.

(If we're keeping this based on the `icons.json` file, I would just make the URL configurable via an `ASSET_ICONS_URL` environment variable, change the ternary condition to use that, and add a default value in `flavors/liquid-mainnet/config.env`.)